docs/src/prototypes/hexagonalAvatar/CustomAvatar.tsx

```diff
@@ -0,0 +1,25 @@
+import * as React from 'react'
+import { Avatar as StardustAvatar, AvatarProps, callable } from '@stardust-ui/react'
+
+const Avatar = (props: AvatarProps & { hexagonal?: boolean }) => {
+  const { hexagonal, variables: propVariables, ...rest } = props
+  const variables = siteVars => {
+    return {
+      ...callable(propVariables)(siteVars),
+      ...(hexagonal && {
```

Comment:
Are you sure with the order of `propsVariables` and the rest? I would say props variables should win.